libs/elements/layout/src/lib/pages/elewa-group-main-page/elewa-group-main-page.component.html

```diff
@@ -7,6 +7,13 @@
     <router-outlet></router-outlet>
   </div>
   <div class="footer">
-    <!-- <elewa-group-footer></elewa-group-footer> -->
+    <elewa-group-footer>
+      <div footer class="footer-content">
+        <h4>do you want to be part of our mission?</h4>
+        <div class="mission">
+          <elewa-group-button-with-animation [message]="'Become an investor'" [mode]="'dark'" [action]="'/'"></elewa-group-button-with-animation>
+        </div>
+      </div>
```

Comment:
very good job but please move this to features/components/banners

call the component footer-mission